posts/2019-09-26-Rust-1.38.0.md

```diff
@@ -95,8 +95,8 @@ Additionally, these functions have been stabilized:
 
 - [`<*const T>::cast`] and [`<*mut T>::cast`]
 - [`Duration::as_secs_f32`] and [`Duration::as_secs_f64`]
-- [`Duration::div_duration_f32`] and [`Duration::div_duration_f64`]
-- [`Duration::div_f32`] and [`Duration::div_f64`]
+- EDIT: A Previous version of this post mistakenly marked these as stable.
```

Comment:
As @Mark-Simulacrum noted, no need to retain the "historical record" here -- just remove the incorrect item from the list.

Reply:
I'm not quite sure what you mean, I did remove it. I think your looking at the older commit maybe? Sorry, I'm new to github.